src/test/cargo-rmc/simple-extern/test_sum.expected

```diff
@@ -0,0 +1 @@
+[external_c_assertion.assertion.1] line 13 assertion rust_add1(x) == x + 1: SUCCESS
```

Comment:
Why do we need this change?

Reply:
Now that we can configure `cargo rmc` with the toml file, we are able to set the `c-lib` flag and actually make this test work, so I moved it from `.ignore` to `.expected`.